scripts/DataStormUtil.py

```diff
@@ -39,6 +39,7 @@ def getEffectiveProps(self, current, props):
             # Default properties for tests that don't specify any DataStorm.Node.* properties
             props.update(
                 {
+                    "DataStorm.Node.Multicast.Enabled": 0,
```

Comment:
This is to fix a configuration bug, introduced from the port to Ice 3.8. We were running tests with both Multicast enabled and Server endpoints. In the original zeroc-ice/datastorm repository we enabled one or the other with the `--multicast` option. 

With this PR I added separate configurations for each case, so we always run both.